ublox_msgs/include/ublox_msgs/serialization.hpp

```diff
@@ -893,6 +893,113 @@ struct UbloxSerializer<ublox_msgs::msg::CfgTMODE3_<ContainerAllocator> > {
     stream.next(m.reserved3[7]);
   }
 };
+///
+/// @brief Serializes the CfgVALDEL message which has a repeated block.
+///
+template <typename ContainerAllocator>
+struct UbloxSerializer<ublox_msgs::msg::CfgVALDEL_<ContainerAllocator> > {
+static void read(const uint8_t *data, uint32_t count,
```

Comment:
Style: can you make this `inline static` like the rest of the `read` methods in this file.

Reply:
Also style: indent these blocks so that they look like part of the `struct`.